### PR TITLE
Tal/test scripts

### PR DESCRIPTION
**Pull request by taoliseki**
Added perf/stress test scripts and files to capture any concurrent connection issues against Kestrel + IIS.

### CODE REVIEW

File: test/Reliability/ConnectionLoadTest/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Run test
+
+Spin up 2 lab machines or Azure VMs. One machine will be generating the contant load and the other one generating the bursty load.
+
+Command to kick off wcat test: wcatrun.bat
+
+We should run the 2 scenarios belows at the same time:
+
+- Constant load example (in wcatrun.bat): 
+  .\wcctl.exe -t .\scenario-contant-load.ubr -f .\settings.ubr -p 80 -c 1 -v 10000 -o .\out.xml -x
```
Comment (SajayAntony) [inline test/Reliability/ConnectionLoadTest/README.md]: Format commands using markdown enclose using \`

**Comment by SajayAntony** [comment]
Remove binary files.
Just point to install instructions for WCAT if you want.

**Comment by SajayAntony** [comment]
:watch:

File: test/Reliability/ConnectionLoadTest/README.md (hunk 1 above)
Comment (taoliseki) [inline test/Reliability/ConnectionLoadTest/README.md]: Fixed

**Comment by SajayAntony** [comment]
:rocket: 
please squash.

**Comment by SajayAntony** [comment]
Please merge only after you sign the CLA.

**Pull request merged**